client/src/components/Notary/NotarizationTable.tsx

```diff
@@ -154,15 +154,15 @@ export const NotarizationTable: React.FC<NotarizationTableProps> = ({
         case NotaryVerificationCircuit.REGISTRATION_MULTICURRENCY_ID:
           return {
             detected_copy: 'The following transaction receipt was detected from your Wise account',
-            metadata_copy: `€${selectedRequest.metadata} EUR on ${selectedRequest.date}`,
+            metadata_copy: `${selectedRequest.subject} on ${selectedRequest.date}`,
             metadata_type_copy: 'history transaction receipt',
             transaction_type_copy: 'depositor registration'
             };
   
         case NotaryVerificationCircuit.TRANSFER:
           return {
             detected_copy: 'The following transaction was detected from your Wise account history',
-            metadata_copy: `€${selectedRequest.metadata} EUR on ${selectedRequest.date}`,
+            metadata_copy: `${selectedRequest.subject} on ${selectedRequest.date}`, // TODO: update this
```

Comment:
Move parsing for `on` or `at` into `parseTimestamp`

Reply:
https://github.com/zkp2p/zk-p2p/pull/354/commits/4b5bd8e7ec570c56771cd0e8644fb31c10e33d8b